### DISCUSSION TITLE
Error: ModuleNotFoundError: No module named 'airflow'  | Docker Image Windows WSL2

### DISCUSSION THREAD

**Discussion by anilkulkarni87** [apache/airflow#14110] [Q&A]
Hi,
I am trying to setup local development with Airflow and was using the Docker-compose.yaml file in the repo. I am encountering the below error when I issues the command `docker-compose up airflow-init` 

Starting airflowdocker_redis_1    ... done
Starting airflowdocker_postgres_1 ... done
Starting airflowdocker_airflow-init_1 ... done
Attaching to airflowdocker_airflow-init_1
airflow-init_1       | DB_BACKEND=postgresql+psycopg2
airflow-init_1       | DB_HOST=postgres
airflow-init_1       | DB_PORT=5432
airflow-init_1       |
airflow-init_1       | Traceback (most recent call last):
**airflow-init_1       |   File "/home/airflow/.local/bin/airflow", line 5, in <module>
airflow-init_1       |     from airflow.__main__ import main
airflow-init_1       | ModuleNotFoundError: No module named 'airflow'
airflowdocker_airflow-init_1 exited with code 1**

Any pointers as to what is causing this?

**Comment by 7yl4r** [comment]
[Relevant stack overflow q/a](https://stackoverflow.com/questions/66791752/no-module-named-airflow-when-initializing-apache-airflow-docker/67026607#67026607).
